compose/integrations/composable-test-cases/testcases/inheritance/composableInterface/main/src/commonTest/kotlin/CollectionOfComposablesTests.kt

```diff
@@ -86,6 +86,44 @@ class CollectionOfComposablesTests {
             actual = root.dump()
         )
     }
+
+    /** Composable lambdas nested more than single level deep in generic types provoke compilation failure for K/Native
+     * https://github.com/JetBrains/compose-multiplatform/issues/3466
```

Comment:
Is there a PR with a fix? The test looks good. I think we can merge it after the fix is merged.

Reply:
forgot to do a pr and pushed it directly to compiler-integration branch first..
https://github.com/JetBrains/compose-multiplatform-core/pull/1141